Standards/scs-XXXX-vN-taxonomy-of-failsafe-levels.md

```diff
@@ -0,0 +1,254 @@
+It may be used to get general hints and directions though.
+
+:::
+
+### Differentiation between failsafe levels and high availability, disaster recovery, redundancy and backups
+
+The levels of failsafeness defined in this decision record classify the possibilities and impacts of failure cases (such as data loss) and the possible measures.
+High Availability, disaster recovery, redundancy and backups are all measures that can and should be applied to IaaS and KaaS deployments by both CSPs and Users to reduce the possibility and impact of data loss.
+So with this document every reader can see to what level of failsafeness their measures protect user data.
+
+To differentiate also between the named measures the following table can be used:
+
+| Term               | Explanation                                                                                                                                                                        |
+| ------------------ | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| High Availability  | Refers to the availability of resources over an extended period of time unaffected by smaller hardware issues. E.g. achievable through having several instances of resources.      |
+| Disaster Recovery  | Measures taken after an incident to recover data, IaaS resource and maybe even physical resources.                                                                                 |
+| Redundancy         | Having more than one (or two) instances of each resource, to be able to switch to the second resource (could also be a data mirror) in case of a failure.                          |
+| Backup             | A specific copy of user data, that presents all data points at a givne time. Usually managed by users themself, read only and never stored in the same place as the original data. |
+
+### Failsafe Levels and RTO
+
+As this documents classifies failure case with very broad impacts and it is written in regards of mostly IaaS and KaaS, there cannot be one simple RTO set.
+It should be taken into consideration that the RTO for IaaS and KaaS means to make user data available again through measures within the infrastructure.
+But this will not be effective, when there is no backup of the user data or a redundancy of it already in place.
+The different failsafe levels, measures and impacts will lead to very different RTOs.
+For example a storage disk that has a failure will result in an RTO of 0 seconds, when the storage backend uses internal replication and still has two replicas of the user data.
+While in the worst case of a natural disaster, most likely a severe fire, the whole deployment will be lost and if there were no off-site backups done by users there will be no RTO, because the data cannot be recovered anymore.
```

Comment:
Maybe the usage of RTO was meant differently by the writer here. In general, RTO is a set time frame (based on risk analysis or BCM), not a measurement of things that happen. What more likely fits here in this paragraph is **MTTR**, mean time to repair.

Reply:
I rephrased the paragraph, could you look over it again please?

Reply:
:+1: